src/main/scala/za/co/absa/commons/config/ConfigurationImplicits.scala

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2020 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.commons.config
+
+import org.apache.commons.configuration.Configuration
+import org.apache.commons.lang.StringUtils.isNotBlank
```

Comment:
I would use `apache.commons.lang3.StringUtils`

Reply:
Again, it's bound to the "old" commons. Since `config V1` depends on `lang V1`, and these utils are for `config V1` then using `lang V3` would mean introducing a new dependency.